Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+            var labelFrame: CGRect = label.frame
+            labelFrame.size = labelSize
+            labelFrame.origin.x = kPadding
+            labelFrame.origin.y = previouslabelFrame.maxY
+            label.frame = labelFrame
+            previouslabelFrame = labelFrame
+        }
+        
+        var totalItemTitleSize: CGSize = totalItemTitle.sizeThatFits(CGSize(width: width - kPadding * 4.0, height: .greatestFiniteMagnitude))
+        totalItemTitleSize.height = totalItemTitleSize.height * resizePercentage * 2
+        var totalItemTitleFrame: CGRect = totalItemTitle.frame
+        totalItemTitleFrame.size = totalItemTitleSize
+        totalItemTitleFrame.origin.x = kPadding
+        totalItemTitleFrame.origin.y = previouslabelFrame.maxY
+        totalItemTitle.frame = totalItemTitleFrame
+        
+        var totalItemSize: CGSize = totalItem.sizeThatFits(CGSize(width: width - kPadding * 4.0, height: .greatestFiniteMagnitude))
+        totalItemSize.height = totalItemSize.height * resizePercentage
+        var totalItemFrame: CGRect = totalItem.frame
+        totalItemFrame.size = totalItemSize
+        totalItemFrame.origin.x = kPadding
+        totalItemFrame.origin.y = totalItemTitleFrame.maxY
+        totalItem.frame = totalItemFrame
+        
+        var textFieldFrame = textField?.frame ?? CGRect.zero
+        var maxY =  totalItemFrame.maxY
+        if let textField = textField {
+            textFieldFrame.size.width = width - kPadding * 2
+            textFieldFrame.size.height = 35
+            textFieldFrame.origin.x = kPadding
+            textFieldFrame.origin.y = maxY + kPadding
+            textField.frame = textFieldFrame
+            maxY = textFieldFrame.maxY
+        }
+        
+        var containerViewFrame: CGRect = containerView.frame
+        containerViewFrame.size.width = width
+        containerViewFrame.size.height = rint(maxY + kPadding * 1.5 * resizePercentage)
+        containerView.frame = containerViewFrame
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        updateSubviews()
+    }
+    
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func showWithType(showType: PopupViewShowType) {
+        super.showWithType(showType: showType)
+        
+        textField?.becomeFirstResponder()
+    }
+}
+
+extension PaymentHandlerPopupView: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        textField.resignFirstResponder()
+        return true
+    }
+}
+
+extension Strings {
+    public static let orderSummary = NSLocalizedString("orderSummary", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Order Summary", comment: "Title for display items")
+    
+     public static let total = NSLocalizedString("total", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Total", comment: "Title for total")
```

Comment:
Needs better comment for translators:

e.g.
`comment: "Title that shows the total cost of an online order form."`